node/lib/util/checkout.js

```diff
@@ -101,13 +102,14 @@ const loadSubmodulesToCheckout = co.wrap(function *(repo, commit) {
 
     const result = {};
     const subFetcher = new SubmoduleFetcher(repo, commit);
-    yield open.map(co.wrap(function *(name) {
+    const doSub = co.wrap(function *(name) {
         const subRepo = yield SubmoduleUtil.getRepo(repo, name);
         const sha = shas[name];
         yield subFetcher.fetchSha(subRepo, name, sha);
         const commit = yield subRepo.getCommit(sha);
         result[name] = { repo: subRepo, commit: commit };
-    }));
+    });
+    yield DoWorkQueue.doInParallel(open, doSub, 30);
```

Comment:
OK, definitely a global for this.

Reply:
It's actually got a reasonable default; I don't know why I'm copy and pasting 30 all over.